akka-actor/src/main/java/akka/japi/pf/ReceiveBuilder.java

```diff
@@ -130,6 +130,23 @@ public boolean defined(Object o) {
     return matchUnchecked(type, predicate, apply);
   }
 
+  /**
+   * Add a new case statement to this builder.
+   *
+   * @param type
+   *          a type to match the argument against
+   * @param externalPredicate
+   *          a external predicate that will be evaluated if the type matches
+   * @param apply
+   *          an action to apply to the argument if the type matches and the
+   *          predicate returns true
+   * @return a builder with the case statement added
+   */
+  public <P> ReceiveBuilder match(final Class<P> type, final java.util.function.BooleanSupplier externalPredicate,
```

Comment:
Not sure this really is worth it. We already have `FI.TypedPredicate` which you can simply use as `_ -> false` done

Reply:
it's original your issue :)

Reply:
lol :) 
I guess it'd be ok to add, answered without reading the ticket... :)